pkg/daemon/daemon.go

```diff
@@ -312,6 +325,14 @@ func ptp4lCreateCmd(nodeProfile *ptpv1.PtpProfile, confFilePath string) *exec.Cm
 func cmdRun(p *ptpProcess, stdoutToSocket bool) {
 	var c net.Conn
 	done := make(chan struct{}) // Done setting up logging.  Go ahead and wait for process
+	defer func() {
```

Comment:
Is everything outside of this defer func just style changes, or necessary for this fix?
If it's not part of this fix, can you put it in a separate commit?

Reply:
This is part of the restart process and I want to backport this once it is thoroughly tested by QE.
So it is good to keep it in one PR to track the fix or for any issues to reverse the changes.

Reply:
Can you split it into two separate commits within the same PR?

Reply:
I don't get it , what two PR?